src/dune/exe.ml

```diff
@@ -194,7 +196,7 @@ let build_and_link_many ~programs ~linkages ~promote ?link_flags cctx =
         link_js ~name ~cm_files ~promote cctx
```

Comment:
The C stubs are ignored when compiling to JS; not sure what we want to do here.

Reply:
One thing we *may* do is warn the user to provide a .js runtime file when they're trying to build the js. I'm not sure if this is the right move however.